components/DoveadmComponent.vue

```diff
@@ -18,115 +18,129 @@ const d = Object.fromEntries(Object.entries(data.doveadm).filter(([k, v]) =>
 ).sort())
 </script>
 
+<style scoped>
+.doveadmList h3:first-of-type {
+  border-top-width: 0;
+  padding-top: 0;
+}
+.doveadmList h3 {
+  border-top: 1px solid var(--vp-c-divider);
+  margin-top: 18px;
+  padding-top: 18px;
+}
+</style>
+
 <template>
- <template v-for="(v, k) in d">
-  <h3 :id="k" tabindex="-1">
-   <code>{{ k }}</code>
-   <a class="header-anchor" :href="'#' + k"></a>
-  </h3>
+ <span class="doveadmList">
+  <template v-for="(v, k) in d">
```

Comment:
Similar to the `EventsComponent`:
- Use `<section>` instead of `<span>`, and
- use `<article>` instead of (inner) `<template>`.

And adapt the styles to say:
```css
.doveadmList article:first-of-type {
  /* ... */
}
.doveadmList article {
  /* ... */
}
```

Reply:
span changed to section

However, like #1000 , template is the correct usage here, since we need to loop between the parent container and the h3 sections, and we don't want anything output to the page.

Reply:
I replied to your reply, let's keep this open until a solution can be decided on.